### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -28,18 +28,23 @@ generate = [
...
     "GtkSource.SmartHomeEndType",
     "GtkSource.SpaceDrawer",
+    "GtkSource.SpaceLocationFlags",
+    "GtkSource.SpaceTypeFlags",
     "GtkSource.StyleScheme",
     "GtkSource.StyleSchemeManager",
     "GtkSource.StyleSchemeChooser",
     "GtkSource.Tag",
     "GtkSource.UndoManager",
+    "GtkSource.ViewGutterPosition",
```
Comment (EPashkin) [inline Gir.toml]: Generated but unused.

File: src/auto/space_drawer.rs [hunk 2]
```diff
@@ -91,10 +95,12 @@ impl<O: IsA<SpaceDrawer> + IsA<glib::object::Object>> SpaceDrawerExt for O {
     //    unsafe { TODO: call ffi::gtk_source_space_drawer_get_matrix() }
     //}
 
-    //#[cfg(any(feature = "v3_24", feature = "dox"))]
-    //fn get_types_for_locations(&self, locations: /*Ignored*/SpaceLocationFlags) -> /*Ignored*/SpaceTypeFlags {
-    //    unsafe { TODO: call ffi::gtk_source_space_drawer_get_types_for_locations() }
-    //}
+    #[cfg(any(feature = "v3_24", feature = "dox"))]
```
Comment (EPashkin) [inline src/auto/space_drawer.rs]: bind_matrix_setting here seems can be generated after manual items from gio

Comment (EPashkin) [reply src/auto/space_drawer.rs]: and "GLib.Variant" seems can be manual too for "get_matrix"

File: Gir.toml (hunk 1 above)
Comment (EPashkin) [reply Gir.toml]: I even don't see it usage in GtkSource-3.0.gir file

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
👍 
But maybe better wait for https://github.com/gtk-rs/gio/pull/69 and InputStream

**Comment by GuillaumeGomez** [comment]
I was wondering about it. Let's wait for it then.

**Pull request merged**